src/Referent/ManagedInstitutionalEventsExporter.php

```diff
@@ -19,6 +27,21 @@ public function exportAsJson(array $managedInstitutionalEvents): string
                 'category' => $institutionalEvent->getCategoryName(),
                 'postalCode' => $institutionalEvent->getPostalCode(),
                 'organizer' => $institutionalEvent->getOrganizer()->getPartialName(),
+                'invitationsCount' => $institutionalEvent->getInvitationsCount(),
+                'edit' => [
+                    'label' => "<span class='btn btn--default'><i class='fa fa-edit'></i></span>",
+                    'url' => $this->urlGenerator->generate(
+                        'app_referent_institutional_events_edit',
+                        ['uuid' => $institutionalEvent->getUuid()]
```

Comment:
do we need to generate absolute URL here (with the host)?

Reply:
No, absolute path is enough :wink: